### DISCUSSION TITLE
How much does bootstrap cost for one year (free version)?

### DISCUSSION THREAD

**Discussion by tvtoanit88** [twbs/bootstrap#40881] [Q&A] [answered]
I want to use bootstrap free version for my project, commercial project, only use features, not sell source code for example build  website-build. Please tell me the price. Because I use. Jquery they say free but actually they do not allow and request to remove. So I ask bootstrap free version how much?

**Comment by julien-deramond** [comment, marked as answer]
Hello @tvtoanit88 
Bootstrap is completely free to use, even for commercial projects, under the [MIT License](https://github.com/twbs/bootstrap/blob/main/LICENSE).
More info in our [Docs > License FAQs](https://getbootstrap.com/docs/5.3/about/license/).

**Reply by tvtoanit88** [reply]
Hello @julien-deramond .
Thank you so much. Sorry, I'm late in replying.
